packages/tokens/bin/lib/tokens/icons.ts

```diff
@@ -0,0 +1,61 @@
+import {FileNodesResponse, Frame} from 'figma-js';
+import {readJson} from 'fs-extra';
+import {optimize, OptimizedSvg} from 'svgo';
+
+import {getPage} from '../figma';
+import {Token, TokenGroup} from '../formats/token';
+import {generateTokens} from '../generateTokens';
+import {PagesId} from '../mappings';
+
+const cleanSvgMarkup = (name: string, svgMarkup: string): string => {
+    const result = optimize(svgMarkup, {
+        multipass: true,
+        plugins: [
+            {name: 'removeViewBox', active: false},
+            {name: 'removeDimensions', active: true},
+            {
+                name: 'removeAttributesBySelector',
+                active: true,
+                params: {selector: "[stroke='#282829']", attributes: ['stroke']} as any,
+            },
+            {
+                name: 'addAttributesToSVGElement',
+                active: true,
+                params: {attribute: {stroke: '#282829'}} as any,
```

Comment:
Should we pivot to `currentColor`?
```suggestion
                params: {attribute: {stroke: 'currentColor'}} as any,
```

Reply:
I like it, I didn't know it existed 👍 . I will also target `[stroke]` as attribute to remove in case someone don't use `#282829` when drawing the icon.